v20.2/performance.md

```diff
@@ -15,44 +15,52 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
 
 CockroachDB has **no theoretical scaling limit** and, in practice, can achieve near-linear performance at 256 nodes. Because the TPC-C results above reflect leaps in scale, to test linear scaling, Cockroach Labs ran a simple benchmark named KV 95 (95% point reads, 5% point writes, all uniformly distributed) on AWS `c5d.4xlarge` machines:
 
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update this chart with latest data - see <https://github.com/cockroachdb/docs/issues/8212>
+
 <img src="{{ 'images/v20.2/linearscale.png' | relative_url }}" alt="CRDB Linear Scale" style="max-width:100%" />
 
 This chart shows that adding nodes increases throughput linearly while holding p50 and p99 latency constant. The concurrency for each scale was chosen to optimize throughput while maintaining an acceptable latency and can be observed in the table below.
 
-| Number of Nodes | Workers | Concurrency
-|-----------------|---------|------------|
-|       16        |    2    |   512
-|       32        |    4    |   512
-|       64        |    4    |   1024
-|       128       |    8    |   1024
-|       256       |    8    |   2048
+| Number of Nodes | Workers | Concurrency |
+|-----------------+---------+-------------|
+|              16 |       2 |         512 |
+|              32 |       4 |         512 |
+|              64 |       4 |        1024 |
+|             128 |       8 |        1024 |
+|             256 |       8 |        2048 |
+
 ## Throughput
 
 As mentioned above, Cockroach Labs believes TPC-C provides the most realistic and objective measure for OLTP throughput. In the real world, applications generate transactional workloads that consist of a combination of reads and writes, possibly with concurrency and likely without all data being loaded into memory. If you see benchmark results quoted in QPS, take them with a grain of salt, because anything as simple as a “query” is unlikely to be representative of the workload you need to run in practice.
 
 With that in mind, however, you can use [Sysbench](https://github.com/akopytov/sysbench) for straight-forward throughput benchmarking. For example, on a 3-node cluster of AWS `c5d.9xlarge` machines across AWS’s `us-east-1` region (availability zones `a`, `b`, and `c`), CockroachDB can achieve 118,000 inserts per second on the `oltp_insert` workload and 336,000 reads per second on the `oltp_point_select` workload. We used a concurrency of 480 on the `oltp_insert` workload and a concurrency of 216 on the `oltp_point_select` workload to generate these numbers.
 
-<img src="{{ 'images/v20.2/sysbench-throughput.png' | relative_url }}" alt="Sysbench Throughput" style="max-width:100%" />
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update this chart with latest data - see <https://github.com/cockroachdb/docs/issues/8212>
 
+<img src="{{ 'images/v20.2/sysbench-throughput.png' | relative_url }}" alt="Sysbench Throughput" style="max-width:100%" />
```

Comment:
Is there some updated data for these Sysbench runs that can be used to update this chart?

I can take a stab at updating the chart but I'm not sure where / how to get the latest data.